ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/config/DownstreamTlsOptions.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.config;
+
+import java.util.Optional;
+
+public interface DownstreamTlsOptions {
```

Comment:
nit: as said prior - if we make this generically named, then maybe we can reuse?

Reply:
renamed to ClientTlsOptions